### PR TITLE
add russian translation

### PR DESCRIPTION
**Pull request by Klucherev**

### CODE REVIEW

File: docs/Translation.md [hunk 1]
```diff
@@ -75,6 +75,7 @@ You can find translation packages for the following languages:
 - French (`fr`): [marmelab/ra-language-french](https://github.com/marmelab/ra-language-french)
 - English (`en`): [marmelab/ra-language-english](https://github.com/marmelab/ra-language-english)
 - Slovak (`sk`): [zavadpe/ra-language-slovak](https://github.com/zavadpe/ra-language-slovak)
+- Russian (`ru`): [klucherev/ra-language-russian](https://github.com/klucherev/ra-language-russian)
```
Comment (djhi) [inline docs/Translation.md]: Please add it in alphabetical order :)

**Comment by Klucherev** [comment]
added in alphabetical order :)

**Comment by djhi** [comment]
It looks like the file was updated before you submitted your pull request. Can you rebase please ?

File: docs/Translation.md [hunk 2]
```diff
@@ -73,10 +73,11 @@ const App = () => (
 You can find translation packages for the following languages:
 
 - Dutch (`nl`): [pimschaaf/ra-language-dutch](https://github.com/pimschaaf/ra-language-dutch)
-- English (`en`): [marmelab/ra-language-english](https://github.com/marmelab/react-admin/tree/master/packages/ra-language-english)
-- French (`fr`): [marmelab/ra-language-french](https://github.com/marmelab/react-admin/tree/master/packages/ra-language-french)
+- English (`en`): [marmelab/ra-language-english](https://github.com/marmelab/ra-language-english)
```
Comment (djhi) [inline docs/Translation.md]: Why did you change this ?

File: docs/Translation.md [hunk 3]
```diff
@@ -73,10 +73,11 @@ const App = () => (
...
+- French (`fr`): [marmelab/ra-language-french](https://github.com/marmelab/ra-language-french)
```
Comment (djhi) [inline docs/Translation.md]: Why did you change this ?

**Comment by fzaninotto** [comment]
Bump, please revert unrelated changes

**Comment by fzaninotto** [comment]
No news for some time, closing
